### DISCUSSION TITLE
Jan 24

### DISCUSSION THREAD

**Discussion by hanseulhee** [GymHubCommunity/GymHub-FE#39] [DailyMeeting]
### ⚽️ 회의 내용

- [x] 개발 현황
- [x] 공유할 내용

### 🏀 오늘 할 일

슬희 - react hook form 세팅 끝내기
수 - wiki 마무리, jotai 설정, npm ci 마무리
수영 - mix penal 공부 중 (ga, hotgar 이미 함)
윤혁 - atom 설정 (페이지까지 만들어야 함 ..) 하다가 빡침 💢

* atom 에서 `template - pages` 를 나눠야 할까??
  -> 찬성 (수영, 수, 윤혁, 슬희)

* SCSS vs module.scss
  -> module 사용 !! 

* Oauth -> 백에서 ??

### 🎱 TMI

슬희 - (왜 안 오심 !) 운동을 많이 해서 늦게까지 공부 함 🤔
수 - **사케** >>>> 중국 술 > 소주 > 양주 🍺
수영 - again cafe...☕ 아침 운동 스케줄 시작 💪
윤혁 - 이과두주 jmt 🍶

**Comment by hanseulhee** [comment]
쏘리쏘리 ~~

**Comment by ngsy** [comment]
프로젝트 끝나면 다 같이 사케 한번 도전 해보시죠!
